templates/contactDetails.html

```diff
@@ -13,7 +13,9 @@
 	<button ng-click="ctrl.deleteContact()" class="icon-delete-white" title="Delete"></button>
 </header>
 <section>
-	<detailsItem ng-repeat="prop in ctrl.singleProperties" name="prop.name" data="prop.data" model="ctrl" index="prop.index" ng-class="[ 'details-item-' + prop.name ]"></detailsItem>
+	<div ng-repeat="(propName, prop) in ctrl.contact.props">
```

Comment:
@Henni as far as I can tell this nested repeat is responsible for not updating the ui after deleting a field ...